### PR TITLE
Contribute

### PR DESCRIPTION
**Pull request by truongtoanbk**
add section and finish tutorial on the youtube

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,11 +118,11 @@ <h1 class="text-capitalize">
           <div class="cart-item d-flex justify-content-between text-capitalize my-3">
             <img src="img-cart/doughnut-1.jpeg" alt="" class="img-fluid rounded-circle" id="item-img">
             <div class="item-text">
-              <p id="cart-item-title" class="font-weight-bold mb-0">cart item</p>
+              <p class="cart-item-title font-weight-bold mb-0">cart item</p>
               <span>$</span>
-              <span id="cart-item-price" class="cart-item-price mb-0">1.99</span>
+              <span class="cart-item-price mb-0">1.99</span>
             </div>
-            <a href="#" class="cart-item-remove" id="cart-item-remove">
```
Comment (truongtoanbk) [inline index.html:125]: too more id

File: js/app.js [hunk 2]
```diff
@@ -1,5 +1,64 @@
-document.getElementById("cart-info").addEventListener("click", function() {
-  const cart = document.getElementById("cart");
-  cart.classList.toggle("show-cart");
-  console.log(cart);
-});
```
Comment (truongtoanbk) [inline js/app.js:5]: remove initial section

**Review by truongtoanbk — COMMENTED** [review]
this is something i made

File: js/app.js (hunk 2 above)
Comment (truongtoantt) [reply js/app.js:5]: ok thank you

**Pull request merged**